packages/reactivue/src/component.ts

```diff
@@ -70,10 +80,10 @@ export const unmountInstance = (id: number) => {
    * instance id unlike the hmr updated components.
    */
   if (isDev) {
-    setTimeout(unmount, 1000)
+    setTimeout(async() => _vueState[id]?.isUnmounting && unmount(id), 1)
```

Comment:
Nice one! Would `setTimeout(fn, 0)` works?

Reply:
Actually that timer was shouldn't even be here like previous commit. I forget it here while testing 😅